README.md

```diff
@@ -1,16 +1,271 @@
+
+In this repository, you will find HELM charts for each FOLIO module.
+
+## Structure
+
+```markdown
+folio-helm-v2/
+├── README.md
+├── example/                        # module chart scaffold
+│   ├── value.yaml                  # library entry point
+│   ├── templates/
+│   │   ├── application.yaml        # main module template model
+│   │   └── NOTES.txt               # chart user instruction model
+│   └── resources                   # resource file examples
+│       ├── ephemeral.properties
+│       └── log4j2.xml
+└── charts/
+    ├── <some-module>/              # FOLIO module HELM chart
+    │   ├── Chart.yaml              # chart information
+    │   ├── values.yaml             # default values for templates
+    │   ├── templates/
+    │   │   ├── application.yaml    # main template
+    │   │   └── NOTES.txt           # chart user instruction
+    │   └── resources/              # chart resource files
+    │       ├── ephemeral.properties
+    │       ├── ..................
+    │       └── log4j2.xml
+    ├── ..................
+    └── folio-common/               # core helpers
+        ├── Chart.yaml
+        ├── values.yaml
+        └── templates/
+            ├── _configmap.tpl
+            ├── _deployment.tpl
+            ├── _helpers.tpl
+            ├── ...............
+            └── _volume.tpl
+```
+
+## Usage
+### Chart usage
+
+The charts in this repository allow users to deploy different modules of the FOLIO project to a Kubernetes cluster using
+the Helm package manager. To do this you can follow the following steps:
+* Search for the particular FOLIO module you are interested in (For example, mod-users).
+* Look at the values.yaml file to understand the default configurations available for this module. Usually,
+they have common [Parameters](#parameters) but some distinctions are possible.
+* Use Helm to install the chart with your unique release name and appropriate configurations.
+
+### Adding/Modifying Charts
+
+For adding new charts or modifying existing ones, this repository is designed to make the process straightforward.
+
+In the example directory, we have a scaffold that you can use as a base for creating new charts.
+
+The key feature in this repository is the `charts/folio-common` directory. This directory contains core chart helpers you need
+to create Kubernetes deployments, services, ingresses, and so on.
+The `folio-common` helpers are included in the `templates/application.yaml` file.
+
+When creating new charts, use the existing pattern and make sure to cater for any unique characteristics
+that the new module might have. But also it give a pattern that needs to be followed.
+
+This approach ensures seamless deployments and maintenance of the FOLIO project modules.
+
+Please DO NOT forget to increase the Chart version if changes were made.
+
+After merging Pull Request to the master branch the job https://jenkins-aws.indexdata.com/job/Rancher/job/Index-helm-charts/
```

Comment:
Please change to https://jenkins-aws.indexdata.com/job/folioRancher/job/folioDevOpsTools/job/indexFolioHelmCharts/

Reply:
Have done.